src/jiraUpdateIssue.gs

```diff
@@ -103,36 +103,28 @@ function updateJiraIssues(headerRow, dataRows) {
  * @param value - the value from the spreadsheet for the specified field
  */
 function formatFieldValueForJira(fieldDefinition, value) {
-  if (fieldDefinition.key == "labels") {
-    if (value == "") {
-      value = null;
-    } else {
-      if (typeof value === 'string' || value instanceof String) {
-        value = value.split(/,\s?/);
-      }
-    }
-    return value;
-  }
   var nullableSchemaTypes = ["number", "date", "user", "array|string", "user", "priority"];
   if (nullableSchemaTypes.indexOf(fieldDefinition.schemaType) >= 0) {
     if (value == "") {
       value = null;
     }
   }
+
+  if (fieldDefinition.schemaType == 'array|string') {
+    if (typeof value === 'string' || value instanceof String) {
+      if (value != null && !isNaN(value)) {
+        // assume if the value is numeric it should be a sprint id 
```

Comment:
Sprints field definition is array|string which basically isn't when pushing back to the API to update issues - it only accepts a single integer id or null *and* the number must not be in quotes in the JSON otherwise there is an error.

Reply:
As per your suggestion a while back I'm going to record the "custom" element of the schema from the Jira response so we can differentiate which field is actually sprint
`schema={custom=com.pyxis.greenhopper.jira:gh-sprint, type=array, items=string, customId=12345}`